types/apis/orders.d.ts

```diff
@@ -235,6 +235,30 @@ export type CreateOrderRequestBody = {
     application_context?: OrderApplicationContext;
 };
 
+export type UpdateOrderRequestBody = {
```

Comment:
Thanks for sending this PR @jocmp! @xuanwan05 can you help review?

Also, we do have this shipping change example to cross-reference: https://github.com/paypal-examples/paypal-sdk-server-side-integration/blob/main/src/order/patch-order.ts#L28

Reply:
Hey thanks for the link @gregjopa. 

I re-pushed the commit to include the body as an array instead of an object based on https://github.com/paypal-examples/paypal-sdk-server-side-integration/blob/93cd4e03c354f3140807ee694e43189509a80c38/src/order/patch-order.ts#L36.